.github/workflows/ci.yml

```diff
@@ -37,3 +37,35 @@ jobs:
 
     - name: Perform CodeQL Analysis
       uses: github/codeql-action/analyze@v1
+
+  release:
+
+    needs: build
+    if: github.ref == 'refs/heads/main'
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up JDK 15
+      uses: actions/setup-java@v1
+      with:
+        java-version: 15
+
+    - name: Cache Local Maven Repo
+      uses: actions/cache@v2.1.2
+      with:
+        path: ~/.m2/repository
+        key: maven-repo
+
+    - name: Release
+      uses: qcastel/github-actions-maven-release@v1.12.16
+      with:
+        release-branch-name: "main"
+        access-token: ${{ secrets.GITHUB_TOKEN }}
+        maven-args: "-DskipTests -DskipITs"
+        maven-repo-server-id: github
+        maven-repo-server-username: ${{ github.actor }}
+        maven-repo-server-password: ${{ secrets.GITHUB_TOKEN }}
+      env:
+        JAVA_HOME: /usr/lib/jvm/java-15-openjdk/
```

Comment:
Why we have to set the JAVA_HOME here? Isn't setup-java doing this already?

Reply:
It should be, but it seems this github action uses its own java environment. I will test if the setup-java is needed here at all.

Reply:
Yes, it does setup a new java environment and therefor needs to have the JAVA_HOME variable being set.